src/App.js

```diff
@@ -2,13 +2,19 @@ import React from 'react';
 import './App.css';
```

Comment:
App .css was imported here, was it used in your code?

Reply:
No it wasn't, let me delete it